aws-codeartifact-domain/src/main/java/software/amazon/codeartifact/domain/CallbackContext.java

```diff
@@ -7,4 +7,5 @@
 @lombok.ToString
 @lombok.EqualsAndHashCode(callSuper = true)
 public class CallbackContext extends StdCallbackContext {
+    private boolean isCreated;
```

Comment:
do we not want to have this always initialized as false?

Reply:
It is implicitly set as false, some other teams add this in a similar way: https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-ecs/blob/8c61884fc5178ad2c2bf0e3a600a6bf44d875418/taskset/src/main/java/com/amazonaws/ecs/taskset/CallbackContext.java#L14

That being said, having it explicitly set here would be a bit more clear, I'll add it